preset/deep-learning/Dockerfile

```diff
@@ -227,7 +227,7 @@ ARG IDEX_VERSION
 ARG DEEPSPEED_VERSION
 
 # PyTorch GPU Env - conda packages
-RUN conda create -yn pytorch-gpu -c intel/label/oneapi -c "${INTEL_CHANNEL}" -c conda-forge \
+RUN conda create -yn pytorch-gpu -c https://software.repos.intel.com/python/conda -c "${INTEL_CHANNEL}" -c conda-forge \
```

Comment:
`intel/label/oneapi` can be removed. Hosted channel does not support labels at the moment. 
It should be
` conda create -yn pytorch-gpu -c "${INTEL_CHANNEL}" -c conda-forge \`